.ci/tav.json

```diff
@@ -0,0 +1,40 @@
+{
+  "versions": [ "19", "18", "16", "14", "12", "10", "8" ],
+  "modules": [
```

Comment:
this looks like it could be the version-framework action. But I understand this might need some modifications in the actions to make it work

Reply:
Oh dear, let me explore that :)

Reply:
Unfortunately, it uses an [internal checkout](https://github.com/elastic/apm-pipeline-library/blob/6aba1f9c90c1bc12dd489a304ae28767776ca7c8/.github/actions/version-framework/action.yml#L21) so we cannot use it unless we change the inputs to support the ref and even though we might need to figure out what's the default behaviour :/

Reply:
https://github.com/elastic/apm-pipeline-library/issues/2171 is tracking this